src/components/symptom-shark/presentational/MonthReport/MonthReport.css

```diff
@@ -0,0 +1,55 @@
+.ss-month-report {
+    position: relative;
+    width: 100%;
+    font-family: ui-rounded, -apple-system, Helvetica, Arial, sans-serif;
+    /* default color for PDF rendering */
+    background: #FFF;
```

Comment:
nit: what is the value of line 7 given the hex?

Reply:
The MonthReport could theoretically be used outside of a PDF and it would reflect dark mode; I wanted to leave that option open basically.  When it's rendered in the browser the var will be used, but the PDF rendered just completely ignores lines with vars so the #FFF will be used